korean/soccer/soccer.md

```diff
@@ -2,7 +2,7 @@
 
 30분 간 열심히 하였지만, 여전히 아무도 찾아오지도 않고 여전히 심심해요.
 
-어떻게 할까요?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.